svelte-responsive-layout/src/ResponsiveImage.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  export let variants = [];
+</script>
+
+<style type="text/postcss">
+  .picture-source {
+    width: 100%;
+    text-align: center;
+
+    source, img {
+      max-width: 300px;
```

Comment:
as a component, this should be agnostic as possible, so the `max-width` should be applied by the context (the parent), so this image can work in other scenarios, where max-width would be a different value.